Action.c

```diff
@@ -585,6 +598,7 @@ void Action_setBindings(Htop_Action* keys) {
    keys['S'] = actionSetup;
    keys['C'] = actionSetup;
    keys[KEY_F(2)] = actionSetup;
+   keys['x'] = actionLslocks;
```

Comment:
actionShowLocks to be consistent with other function names and there is no lslocks used in this (thankfully)

Reply:
Output had no consistend maschine-readable format that would not simply warrant to read `/proc/locks` directly.